modules/enum_glue/main.py

```diff
@@ -1,154 +1,156 @@
+    ######
+
+    all = False
+    if args.connections is False and args.databases is False and args.crawlers is False and args.jobs is False and args.dev_endpoints is False:
+        all = True
+
+    if args.regions is None:
+        regions = get_regions('glue')
+        if regions is None or regions == [] or regions == '' or regions == {}:
+            print('This module is not supported in any regions specified in the current sessions region set. Exiting...')
+            return
+    else:
+        regions = args.regions.split(',')
+
+    all_connections = []
+    all_crawlers = []
+    all_databases = []
+    all_dev_endpoints = []
+    all_jobs = []
+    for region in regions:
+        print('Starting region {}...'.format(region))
+        client = pacu_main.get_boto3_client('glue', region)
+
+        # Connections
+        if args.connections is True or all is True:
+            connections = fetch_glue_data(client, 'get_connections', 'ConnectionList', print)
+            print('  {} connection(s) found.'.format(len(connections)))
+            all_connections += connections
+
+        # Crawlers
+        if args.crawlers is True or all is True:
+            crawlers = fetch_glue_data(client, 'get_crawlers', 'Crawlers', print)
+            print('  {} crawler(s) found.'.format(len(crawlers)))
+            all_crawlers += crawlers
+
+        # Databases
+        if args.databases is True or all is True:
+            databases = fetch_glue_data(client, 'get_databases', 'DatabaseList', print)
+            print('  {} database(s) found.'.format(len(databases)))
+            all_databases += databases
+
+        # Development Endpoints
+        if args.dev_endpoints is True or all is True:
+            dev_endpoints = fetch_glue_data(client, 'get_dev_endpoints', 'DevEndpoints', print)
+            print('  {} development endpoint(s) found.'.format(len(dev_endpoints)))
+            all_dev_endpoints += dev_endpoints
+
+        # Jobs
+        if args.jobs is True or all is True:
+            jobs = fetch_glue_data(client, 'get_jobs', 'Jobs', print)
+            print('  {} job(s) found.'.format(len(jobs)))
+            all_jobs += jobs
+
+    summary_data = {
+        'connections': len(all_connections),
+        'crawlers': len(all_crawlers),
+        'databases': len(all_databases),
+        'dev_endpoints': len(all_dev_endpoints),
+        'jobs': len(all_jobs),
+    }
+
+    glue_data = deepcopy(session.Glue)
+    glue_data['Connections'] = all_connections
+    glue_data['Crawlers'] = all_crawlers
+    glue_data['Databases'] = all_databases
+    glue_data['DevEndpoints'] = all_dev_endpoints
+    glue_data['Jobs'] = all_jobs
+    session.update(pacu_main.database, Glue=glue_data)
+
+    print('\n{} completed.\n'.format(module_info['name']))
+    return summary_data
+
+
+def summary(data, pacu_main):
+    out = ''
+    for key in data:
+        out += '  {} total {}(s) found.\n'.format(data[key], key[:-1])

```

Comment:
ideally we only print what was enumerated, so if --connections was passed in, only show enumerated connections, even if it is 0